scripts/s2i-generate.sh

```diff
@@ -31,14 +37,28 @@ phase "Inspecting context subdirectory '${PARAMS_SUBDIRECTORY}'"
 [[ ! -d "${PARAMS_SUBDIRECTORY}" ]] &&
     fail "Application source code directory not found at '${PARAMS_SUBDIRECTORY}'"
 
+phase "Preparing the environment variables"
+
+# create directory in the specified environment path only if its not already present
+if [[ ! -d "${ENV_DIR}" ]]; then
+    phase "Creating directory '${ENV_DIR}'"
+    mkdir -v -p "${ENV_DIR}"
+fi
```

Comment:
Given we're using a mount point provided to the task, it should not try to create the directory, either it exists or the script should return error.